testdata/get_disabled_user.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Nit: Since this is just about testing a disabled user record, I'd say remove fields like `providerUserInfo` and `mfaInfo`.

Reply:
Does it matter either way?  I simply copied get_user.json and modified it to be disabled.

Reply:
Smaller, uncluttered resource files are easier to understand. Also prevents somebody from using them in test cases that were not originally intended.